src/Psl/Str/Byte/after.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Psl\Str\Byte;
+
+use Psl;
+
+/**
+ * @throws Psl\Exception\InvariantViolationException If the $offset is out-of-bounds.
+ *
+ * @psalm-pure
+ */
+function after(string $haystack, string $needle, bool $include_needle = false, int $offset = 0): string
```

Comment:
Not sure about the `include_needle`. Wont you expect it to be always false because you are searching for a string *after* another string?
What would be the use-case?

Maybe better if you create variants like `after_with_needle` instead of providing a boolean option that makes the function behave differently?

Reply:
i agree with you, i have remove the `$include_needle` argument, and instead of returning the $hystack when $needle is not found, now null is returned.

i don't think `after_with_needle` is needed. as it would make things a bit more complicated, and it's the same as:

```php
$result = Str\after($haystack, $needle);
$result = null === $result ? $needle.$result : null;
```